src/game/editor/editor_history.cpp

```diff
@@ -0,0 +1,65 @@
+#include <engine/shared/config.h>
+
+#include "editor.h"
+#include "editor_actions.h"
+#include "editor_history.h"
+
+void CEditorHistory::RecordAction(std::shared_ptr<IEditorAction> const &pAction)
```

Comment:
```suggestion
void CEditorHistory::RecordAction(const std::shared_ptr<IEditorAction> &pAction)
```

We [use](https://github.com/ddnet/ddnet/blob/03400f6ff384a9c8a64f3ebf26f1003858e362a1/src/game/server/gamecontext.h#L102) `const char *` instead of `char const *`, the same is [applicable](https://github.com/ddnet/ddnet/blob/03400f6ff384a9c8a64f3ebf26f1003858e362a1/src/engine/shared/json.h#L6) for structs/classes/whatever.